### PR TITLE
Put discussion to standard layout on react

### PR DESCRIPTION
**Pull request by RomanValihura**

### CODE REVIEW

File: app/imports/ui/react/discussion/containers/DiscussionContainer/index.js [hunk 1]
```diff
@@ -1,21 +1,46 @@
...
+import { setAt, reset, setDiscussion } from '/client/redux/actions/discussionActions';
+import { Discussions } from '/imports/share/collections/discussions';
+
+const discussionLoad = ({ dispatch, urlItemId }, onData) => {
+  const subscription = DiscussionSubs.subscribe('discussionsByStandardId', urlItemId);
+
+  if (subscription.ready()) {
+    const discussion = Discussions.findOne({ linkedTo: urlItemId });
+    const reduxActions = [
+      setDiscussion(discussion),
```
Comment (RevelsGit) [inline app/imports/ui/react/discussion/containers/DiscussionContainer/index.js]: there is no need to batch a single action.

File: app/imports/ui/react/standards/containers/StandardsDataLoader/index.js [hunk 2]
```diff
@@ -78,14 +78,12 @@ import _counter_ from '/imports/startup/client/mixins/counter';
...
-const loadInitialData = ({
-  dispatch,
-  isDiscussionOpened = false,
-}, onData) => {
+const loadInitialData = ({ dispatch }, onData) => {
   const userId = Meteor.userId();
   const orgSerialNumber = parseInt(FlowRouter.getParam('orgSerialNumber'), 10);
   const filter = parseInt(FlowRouter.getQueryParam('filter'), 10) || 1;
   const urlItemId = FlowRouter.getParam('urlItemId');
+  const isDiscussionOpened = FlowRouter.getRouteName() === 'standardDiscussion' ;
```
Comment (RevelsGit) [inline app/imports/ui/react/standards/containers/StandardsDataLoader/index.js]: it is better to provide this prop via router eg. `mount2(StandardsProvider, { isDiscussionOpened: true });

**Comment by RomanValihura** [comment]
I fixed all you review remarks

**Comment by RevelsGit** [comment]
Messages are never marked as viewed.

**Pull request merged**